intelmq/bots/experts/http/expert_content.py

```diff
@@ -0,0 +1,58 @@
+"""
+HTTP Content Expert Bot
+
+SPDX-FileCopyrightText: 2021 Birger Schacht <schacht@cert.at>
+SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+from typing import List
+
+from intelmq.lib.bot import Bot
+from intelmq.lib.utils import create_request_session
+
+
+class HttpContentExpertBot(Bot):
+    """
+    Test if a given string is part of the content for a given URL
+
+    Parameters
+    ----------
+    field: str
+        The name of the field containing the URL to be checked (defaults to 'source.url').
+    needle: str
+        The string that the content available on URL is checked for.
+    overwrite:
+        Specifies if an existing 'status' value should be overwritten.
+    """
+    field: str = "source.url"  # The field containing the URL
+    needle: str = None
+    overwrite: bool = True
+
+    def process(self):
+        event = self.receive_message()
+
+        if self.field in event and self.needle is not None:
+            self.set_request_parameters()
```

Comment:
Can we move the `set_request_parameters` to the init, as it is static?

Reply:
done